sdk/js/__tests__/EventEmitter.spec.js

```diff
@@ -91,6 +91,33 @@ describe('EventEmitter tests', () => {
         })
     })
 
+    describe('emitVariableEvaluated', () => {
+        it('should emit variable evaluated event if subscribed to variable evaluations', () => {
+            const allUpdatesHandler = jest.fn()
+            const variableKeyHandler = jest.fn()
+            const evaluatedVariable = {
+                _id: 'variable_id',
+                key: 'my-variable-key',
+                value: 'my-new-value',
+                type: 'my-type',
+            }
+            eventEmitter.subscribe('variableEvaluated', allUpdatesHandler)
```

Comment:
should this be `variableEvaluated:*` to match the others?

Reply:
and we should also emit `variableEvaluated:<key>` probably?

Reply:
Yep! On it. Didn’t want to go too deep without your green light.

Reply:
That was way easier than I thought